Fixtures/Miscellaneous/Plugins/SwiftFilePlugin/SimpleSwiftScript.swift

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/swift
```

Comment:
does this support pointing to any executable or is it locked to swift? just checking ... one might prefer to use e.g. bash sometimes.

Reply:
You could use a bash script as a build tool plugin executable and I believe it would work today, without this patch, yes.